src/mailing/sender.js

```diff
@@ -51,7 +51,7 @@ async function sendForgotPasswordEmail(toEmail, resetToken) {
       template: 'forgot_password',
       message: { to: toEmail },
       locals: {
-        resetLink: `${process.env.BACKEND_BASE_URL}/reset_password?token=${resetToken}`,
+        resetLink: `${process.env.BACKEND_BASE_URL_AUTH_PROD}/resetPassword?token=${resetToken}`, //or BACKEND_BASE_URL_AUTH_DEV in development
```

Comment:
Just keep as `BACKEND_BASE_URL` . Prod vs Dev won't require a code change in that case, just a matter of changing the value in the environment variable itself.

Reply:
If you're wondering if the same should be the case for Order.js where we use `DEV_ORDER_EXPIRY_DURATION` and `PROD_ORDER_EXPIRY_DURATION` ; the answer is yeah we could probably simplify that one as well by just having a single environment variable instead :shrug:

Reply:
@akosasante I just live it in .env as` BACKEND_BASE_URL=https://localhost:8000/api/v1/auth ` , right? I was trying to enter to render account to add this new variable- but the credentials are not valid, how can I acces it?

Reply:
Yes exactly :+1: 

Looks like that one expired; try this link (and feel free to save the credentials to your local password manager :smile: ): https://send.bitwarden.com/#wHuUS7Dl8kiRnLBpAQu4eQ/VAIpn8a9nfb6BICMdgJPjA

Reply:
@akosasante sure! thank you!  just created an account on bitwarden to store passwords/ didn't know about this service

Reply:
@akosasante tested it in Postman, and added variables to render.com. can I merge ?